.github/workflows/ci.yml

```diff
@@ -452,6 +464,26 @@ jobs:
             Reports/e2e/traces/*.zip
           retention-days: 30
 
+      - name: Upload pages artifact for GitHub Pages
+        if: always() && github.ref == 'refs/heads/main'  # Only deploy on main branch
```

Comment:
I want it to deploy to gh pages when the PR runs, so that I can view the output.